### DISCUSSION TITLE
about input file

### DISCUSSION THREAD

**Discussion by mimboch** [epochpic/epoch#179] [Q&A] [answered]
Hi all
I checked my input file but I couldn't detect the exact error
Could you help me please?

![220](https://user-images.githubusercontent.com/46644633/111670749-747eed00-8818-11eb-99d5-aadfd6521ac2.png)

[input.txt](https://github.com/Warwick-Plasma/epoch/files/6166118/input.txt)

**Comment by keithbennett** [comment]
The posted input deck does not produce the error shown in the screenshot. Also, this is not a bug report so should be in the issue tracker. Please open such questions in the Q&A discussions section.

**Reply by mimboch** [reply]
maybe the problem with code version!

**Reply by keithbennett** [reply]
No, I don't think so. This is an identical input deck to the one you posted in your previous issue. If it is used with 4.17.13 then you get the exact same error message (unrecognized semicolon) and not the message posted here.

**Reply by keithbennett** [reply]
Sorry, my mistake. I had mixed up the input decks. I will look again.

**Reply by mimboch** [reply]
Thank you very much Keith for your help

**Reply by keithbennett** [reply]
You have a typo on line 122. It should be "particles" not "particle". You also cannot specify "species" for this output variable. The next release will contain the input deck line number in the error message which should help to track down such issues.

**Comment by keithbennett** [comment, marked as answer]
You have a typo on line 122. It should be "particles" not "particle". You also cannot specify "species" for this output variable. The next release will contain the input deck line number in the error message which should help to track down such issues.
